.env.dist

```diff
@@ -0,0 +1,9 @@
+# URLs
```

Comment:
What does the `.dist` extension mean? I think that `.env.sample` or `.env.example` are more common names, never saw this extension before